docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -14,6 +14,8 @@ You can configure the Connector runtime environment in the following ways:
 
 ## Connecting to Zeebe
 
+<!-- NEEDS ATTENTION -->
+
 In general, the Connector Runtime will respect all properties known to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe).
```

Comment:
@sbuettner Can we adjust this to move away from the community-supported project?

Reply:
Yes, as soon as this has been done: https://github.com/camunda/connectors/issues/2896

Reply:
Leaving this for now, will follow up in a separate PR to remove it.